cmd/kro/commands/generate/generate.go

```diff
@@ -33,12 +33,13 @@ type GenerateConfig struct {
 	outputFormat                string
 }
 
-var config = &GenerateConfig{}
+var generateConfig = &GenerateConfig{}
```

Comment:
nit: imo unneeded variable rename

Reply:
let's do these in a seperate PR @DhairyaMajmudar